addons/l10n_mx/data/template/account.tax-mx.csv

```diff
@@ -83,20 +83,20 @@
 "","","","","","","","","","","","","","","","tax","invoice","cuenta118_01","","",""
 "","","","","","","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","","","","","","tax","refund","cuenta118_01","+DIOT: Refunds","",""
-"tax14","20","16%","","VAT(16%)","16.0","percent","purchase","tax_group_iva_16","on_payment","cuenta119_01","","iva","Tasa","","base","invoice","","+DIOT: 16%","IVA(16%) compras","IVA(16%)"
-"","","","","","","","","","","","","","","","tax","invoice","cuenta118_01","","",""
+"tax14","20","16%","","VAT(16%)","16.0","percent","purchase","tax_group_iva_16","on_payment","cuenta119_01","","iva","Tasa","","base","invoice","","+DIOT: 16% VAT Base","IVA(16%) compras","IVA(16%)"
+"","","","","","","","","","","","","","","","tax","invoice","cuenta118_01","+DIOT: 16%","",""

```

Comment:
I think it's not a good idea to reuse a tax tag for the tax amount which was previously used for the base amount because existing AMLs will retain their existing tags.

Instead you can create a new tax tag for the tax amount and edit the tax report to use the new tax tag instead of the old.

Same comment for all the places below where this happens.

Reply:
I agree that the existing taxes will have incorrectly configured tags, but even without moving them, it will be necessary to reconfigure them to add the new tags, so we believe it is better to leave this tag configuration as it is, since it groups the base tags for each tax (creditable and non-creditable), resulting in fewer final tags for the user and reducing the possibility of human error.

Reply:
From the perspective of tax configuration moving tags around is fine, since you can always just reconfigure the taxes.

The problem is with the existing Journal Items (AccountMoveLines) because they will keep their old tags.
If you use a previously base tag as a tax tag, the old journal items that used that tag will not make sense anymore. It complicates database audits.

It's better if we can avoid that by defining new tags - even if it means there are more tags than would have been necessary if this had been written from scratch.

If we want to do that kind of change, we should do it only in master and with an upgrade script that migrates the tags.

Reply:
Oh right, that makes sense, I've already changed the code to use the current labels in the base and the new ones in the tax.